README.md

```diff
@@ -14,6 +14,7 @@
 
 1. Executes notebooks using pytest and nbclient, allowing parallel notebook testing
 2. Optionally writes back to the repo, allowing faster building of [nbsphinx](https://github.com/spatialaudio/nbsphinx) or [jupyter book](https://github.com/executablebooks/jupyter-book) docs
+3. Optionally break down the notebook into individual tests by section (see "by cell" mode)
```

Comment:
I _think_ this broadens the scope of _Who is this for?_ above. Please could you generalise/elaborate on the new use cases we are accommodating on line 11?